bridge_adaptivity/bridge_lti/provider.py

```diff
@@ -67,9 +67,9 @@ def lti_launch(request, collection_id=None):
         return learner_flow(request, lti_consumer, params, collection_id=collection_id)
 
 
-def instructor_flow(request, collection_id=None):
+def instructor_flow(_request, collection_id=None):
```

Comment:
Why do you start function param name with an underscore? It looks odd

Reply:
Tried to stress that var isn't used. There is convention about '_', but in Django it is used as gettext.
So in Edx I saw '__' usage. Will it work?

Reply:
If the parameter is not used why do we need it in the function?